### ISSUE TITLE
Homepage is not reachable, in some private URL

### ISSUE THREAD

**Issue by sanhue0302** [CocoaPods/CocoaPods#2069]
i set my homepage to a  google doc link, and only some people that i allowed can see it.
i can't pass lint, because it is not reachable
is this homepage check necessary for cocoapods
or just check by owner with a independent way like add a parameter?

**Comment by jakemarsh** [comment]
I ran into this just now, I know this issue is closed, but I'd like to add a comment here to explain my use case:

I've created a private spec repo and I'd like to push one of my pods to it. I'm working on a set of pods that (for now at least) need to remain private. I use the URL to the github repo of my pods as the `homepage` in my podspecs.

Only problem is that until I make their repos public on Github, these URLs are inaccessible.

I was able to work around it trivially by simply setting my user page on github as the homepage in the podspecs temporarily, and once I'm ready to release them publicly someday I'll change them to the real thing.

An easy workaround but I've now spent 20 minutes figuring this all out, so fwiw I'd like to suggest removing this requirement or possibly finding a way to skip this check in the linter when pushing to private spec repo.
